ffs/minerselector/sr2/sr2.go

```diff
@@ -0,0 +1,143 @@
+package sr2
```

Comment:
Summary of the selector:
- Fetch the json file
- Parse it
- Get covered on some possibly weird values (amount < 0, etc)
- Use the ask cache index as much as possible. Some miners might not be there if they have 0 power. So instead of changing the ask index to include also miners with 0 power (which are a lot and would affect index building times), I decided that if a miner in this trusted list hasn't a cached ask, we'd do that directly here.